app/models/pipeline.py

```diff
@@ -79,8 +94,13 @@ class Photo(db.Model):
 
     photo_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     album_id = db.Column(db.Integer, db.ForeignKey('albums.album_id', ondelete='CASCADE'), nullable=False)
-    photo_data = db.Column(db.LargeBinary)
+    photo_data = db.Column(db.Text)
```

Comment:
I'm worried this might break my implementation but lmk why you chose text

Reply:
Actually yea it would lmao.  But my reason is the one above with the altered table script with the session memory and stuff. I know you mentioned in the meeting that you were thinking about storing links to photos in the filesystem, and I believe that would be less time-consuming/easier to implement than encoding and decoding photo data.

Reply:
i'll be looking into how to store the photos in an actual file and just retrieve them w/ links later. once i implement that i don't think we'll have to worry about this at all. in the meantime though, as far as making the photo data text instead of bytea, i still don't understand how that affects the cookie storage problem if the cookie just stores a reference to the listing. im sure you've dealt w/ this inside and out tho lol so i'm sure its something i'm just missing

Reply:
I understand what you mean, but the session data in the community route sends data to the user that is saved in their session, such as `post_id` and other stuff like the post content, and the `photo_data` to render the picture on the HTML page, which in this case is a link. Storing large base64 strings or large objects in a session can result in data loss (as I discovered), so I believe it is preferable to retain the text field.

This is not the only method, but it requires only the necessary database lookups and keeps the community page stable throughout the session.